records_mover/db/postgres/unloader.py

```diff
@@ -101,6 +101,11 @@ def known_supported_records_formats_for_unload(self) -> List[BaseRecordsFormat]:
                                    }),
         ]
 
+    def can_unload_to_scheme(self, scheme: str) -> bool:
+        # Unloading is done via streams, so it is scheme-independent
+        # and requires no scratch buckets.
+        return True
```

Comment:
In one movement scenario, we ask a source database to export into a records directory (basically a pile of CSVs with a JSON file for metadata).  This is how we can find out if the source database can export into the right kind of place for the target database (or whatever) to import.